config/crds/faros_v1alpha1_clustergittrackobject.yaml

```diff
@@ -6,6 +6,13 @@ metadata:
     controller-tools.k8s.io: "1.0"
   name: clustergittrackobjects.faros.pusher.com
 spec:
+  additionalPrinterColumns:
+  - JSONPath: .status.conditions[0].status
```

Comment:
What do you think about using a JSONPath expression like this (disclaimer: I have not tested that this works, but I think it should?):
```
.status.conditions[?(@.type=="ObjectInSync")].status
```
rather than relying on the fact that we (currently) only have one condition in the status? (Same would go for GTOs as well).

Reply:
I also preferred this, thank you!